Source/Utilities.H

```diff
@@ -117,8 +118,9 @@ pc_rst_int_e(
       if (
         S(i, j, k, UEINT) <
         (S(i, j, k, URHO) * std::numeric_limits<amrex::Real>::min())) {
-        // const amrex::Real eos_state_rho = S(i, j, k, URHO);
-        const amrex::Real eos_state_T = std::numeric_limits<amrex::Real>::min();
+        const amrex::Real small_num =
```

Comment:
can we pull the definition of `small_num` out of the if conditions here so we only have one declaration for all of these, instead of 3? Another thing we could do is put this into `Constants.H`. There's already a `small_num` there. Maybe a `real_small_num`? haha

Reply:
Agreed.